starwars/src/charList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Character from './character'
+import App from './App'
+
+const CharList = props => {
+
+    return (
+        <div>
+         <h2>Character List</h2>
+         {props.chars.map (char => {
```

Comment:
Nicely implemented map method. Try to keep your spaces consistent throughout your code to keep readability high.